README.md

```diff
@@ -1 +1,14 @@
-# DX_UPD
\ No newline at end of file
+# DX_UPD
+
+All script are made tested using Python 3.6.8
+
+## Installation
+To run the UPD scripts we need to create a virtual python environment
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip install --upgrade pip
+pip install -r requirements.txt
+```
```

Comment:
same comments from https://github.com/UMCUGenetics/Dx_BAF/pull/1 apply here

Reply:
added  .gitignore.

upgrade pip is good practice (/necessary for our HPC)